draft-nygren-httpbis-httpssvc.md

```diff
@@ -441,6 +438,11 @@ load-balancing.
 For a client to construct the equivalent of an Alt-Svc HTTP response header:
 
 1. For each RR, the SvcDomainName MUST be inserted as the uri-host.
+   If SvcDomainName is empty ("") then the RRNAME for the final HTTPSSVC
```

Comment:
This talks about SvcDomainName being empty, but SvcDomainName can't be empty in this serialization.